tests/modules/commandfilter.c

```diff
@@ -89,6 +89,29 @@ int CommandFilter_LogCommand(RedisModuleCtx *ctx, RedisModuleString **argv, int
     return REDISMODULE_OK;
 }
 
+static int brpoplpush_count = 0;
+
+/* Filter to protect against Bug #11894 reappearing */
+void CommandFilter_BlpopFilter(RedisModuleCommandFilterCtx *filter)
```

Comment:
let's rename it (it's for brpoplpush now, not blpop).
let's add a few words in the comment as to what it does to check that bug.

Reply:
done